grakn-graql/src/main/antlr4/ai/grakn/graql/internal/antlr/Graql.g4

```diff
@@ -21,22 +21,29 @@ computeQuery   : 'compute' computeMethod ;
 
 variables      : VARIABLE (',' VARIABLE)* ;
 
-computeMethod  : min | max | median | mean | std | sum | count | path | paths | cluster | degrees ;
+computeMethod  : min | max | median | mean | std | sum | count | path | paths
+               | connectedComponent | kCore | degree | coreness ;
 
 min            : MIN      'of' ofList      ('in' inList)? ';' ;
 max            : MAX      'of' ofList      ('in' inList)? ';' ;
 median         : MEDIAN   'of' ofList      ('in' inList)? ';' ;
 mean           : MEAN     'of' ofList      ('in' inList)? ';' ;
 std            : STD      'of' ofList      ('in' inList)? ';' ;
 sum            : SUM      'of' ofList      ('in' inList)? ';' ;
-degrees        : DEGREES ('of' ofList)?    ('in' inList)? ';' ;
-cluster        : CLUSTER ('of' id    )?    ('in' inList)? ';' clusterParam* ;
+coreness       : CENTRALITY ('of' ofList)? ('in' inList)? ';' 'using k-core' ('where' 'min-k' '=' INTEGER)? ';';
+degree         : CENTRALITY ('of' ofList)? ('in' inList)? ';' 'using degree' ';';
+connectedComponent    : CLUSTER            ('in' inList)? ';' 'using connected-component' ('where' ccParam+)? ';';
+kCore                 : CLUSTER            ('in' inList)? ';' 'using k-core'              ('where' kcParam+)? ';';
```

Comment:
Instead of `'using connected-component'` and `'using k-core'`, make it:

```
'using' 'k-core'
```

That way we're white-space insensitive!
You can add as many spaces/new-lines/comments as you like in between `using` and `k-core`.